src/modules/rest/impl/src/main/java/it/geosolutions/geostore/services/rest/security/HeadersAuthenticationFilter.java

```diff
@@ -77,6 +81,9 @@ protected void authenticate(HttpServletRequest req) {
                     if (groupName.equals(GroupReservedNames.EVERYONE.groupName())) {
                         everyoneFound = true;
                     }
+                    if(groupHeaderPrefix != null && groupName.startsWith(groupHeaderPrefix)) {
+                    	groupName = groupName.replaceAll(groupHeaderPrefix, "");
```

Comment:
I think this is going to transform ROLE_SOMETHINGROLE_ into SOMETHING, because of replaceAll